### ISSUE TITLE
Error in evaluation of a strategy

### ISSUE THREAD

**Issue by Ai360n** [ContinualAI/avalanche-rl#18]
Hello,

I am writing to inform you that unfortunately there is a problem when a scenario will be evaluated after training. In other words, when the below function will be called:

strategy.eval(scenario.test_stream)

This error will be shown:

AttributeError: 'RLScenario' object has no attribute 'test_stream'

Is there any update or solution for this issue?

Thanks in advance

avalanche-lib==0.3.1

**Comment by EliaPiccoli** [comment]
As you can see in the examples folder to make it work correctly you should write:

`strategy.eval(scenario.eval_stream)`

@AntonioCarta Should we stick with this name for the test/evaluation stream or should we change it?

**Comment by AntonioCarta** [comment]
I think `test_stream` is a better default. Most benchmarks have a `train/test` split, not a `train/eval` split.

**Comment by Ai360n** [comment]
> As you can see in the examples folder to make it work correctly you should write:
> 
> `strategy.eval(scenario.eval_stream)`
> 
> @AntonioCarta Should we stick with this name for the test/evaluation stream or should we change it?

Yes, thank you. Pardon me, now, as I save the checkpoints, I get this error during the evaluation when I use  eval_stream:

RuntimeError: Checkpoint file checkpoints/99/checkpoint.pth already exists.

What is the problem? However, I save the checkpoint at end!

**Comment by AntonioCarta** [comment]
We changed the checkpointing mechanism in the avalanche master because it had some issues. It is also untested with avalanche-RL still, so you may have to wait for the next avalanche release for a stable checkpointing system.

**Comment by Ai360n** [comment]
Thank you so much for your reply. Excuse me, when the next version of Avalanche will be release?

Thanks

**Comment by AntonioCarta** [comment]
It's planned for the end of June. Avalanche-rl will support the new version soon after the release.
